config/import-control-test.xml

```diff
@@ -16,4 +18,26 @@
 
   <allow pkg=".*" regex="true" />
 
+  <!-- Till https://github.com/checkstyle/checkstyle/issues/6916 -->
+  <subpackage name="checks">
+    <allow pkg="org.junit" local-only="true"/>
+    <subpackage name="(blocks|coding|design|header|imports|indentation|javadoc)" regex="true">
+      <allow pkg="org.junit"/>
+    </subpackage>
+    <subpackage name="(metrics|modifier|naming|regexp|sizes|whitespace)" regex="true">
+      <allow pkg="org.junit"/>
+    </subpackage>
+  </subpackage>
+  <subpackage name="(ant|api|filefilters|filters|grammar|gui|utils|xpath)" regex="true">
+    <allow pkg="org.junit"/>
+  </subpackage>
+  <subpackage name="internal">
+    <allow pkg="org.junit"/>
+    <allow pkg="org.mockito"/>
+    <allow pkg="org.powermock"/>
```

Comment:
please explain why this is required?
in test area we allow all, and disallow only what we know is not good.

It is complete opposite to main code, where we disalllow all and require explicit allow .... to easily detect new dependencies between packages.

Reply:
Done. Some explanation:

The packages  `powermock` and  `mockito` are forbidden at the root level for tests:
https://github.com/checkstyle/checkstyle/blob/master/config/import-control-test.xml#L14-L15

This leads to a large number of violations in the `internal` package that are suppressed blindly at
https://github.com/checkstyle/checkstyle/blob/master/config/suppressions.xml#L102-L103

It is part of #6439 
See https://github.com/checkstyle/checkstyle/pull/6445/files#diff-3719a986b249cc213b4c5d292aa45f30R101

Working on this issue I made a small config with `ImportControl` check and configured it so that there were no violations at all.

Reply:
Ok, let's do this in separate PR